docs/requirements.md

```diff
@@ -2,88 +2,76 @@
+| Platform (1 replica per component) | 4                       | 8                        |
+| Analysis Worker                    | 5 (per Analysis Worker) | 10 (per Analysis Worker) |
 
-```text
-CPU: workerResources.requests.cpu + (pluginResources.requests.cpu * 4)
+The resources described on the following table are based on our experience and are also the defaults in the [values-production.yaml](https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml) file, which you might need to adapt taking into account your use case.
 
-Memory: workerResources.requests.memory + (pluginResources.requests.memory * 4)
+| Installation type                        | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ---------------------------------------- | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| Kubernetes Small Installation            | 16            | 32 GB          | 1                      | 2                                  | 4              | 8 GB            | 10                     | 20 GB                   |
+| Kubernetes Medium Installation (default) | 32            | 64 GB          | 2                      | 4                                  | 8              | 16 GB           | 20                     | 40 GB                   |
+| Kubernetes Big Installation              | 60+           | 110+ GB        | 2+                     | 10+                                | 8+             | 16+ GB          | 50+                    | 100+ GB                 |
 
-Number of concurrent analysis: workers.dedicatedMax
-```
+**NOTE:**
+For microk8s we added 1.5 CPU and 1.5 GB extra in the "Platform" meant to be used by microk8s itself.
 
-Given the previous values, the total number of resources required should be the "per-analysis" amount times the number of concurrent analysis.
+| Installation type              | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ------------------------------ | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| MicroK8s Minimum               | 16            | 32 GB          | 1                      | 2                                  | 5.5            | 9.5 GB          | 10                     | 20 GB                   |
+| MicroK8s Recommended (default) | 20+           | 32+ GB         | 1-2                    | 2                                  | 11+            | 20+ GB          | 10                     | 20 GB                   |
```

Comment:
Maybe bold out this row so it stands out even more as the default?

Reply:
I tried but it gets a bit confuse with the header:

![image](https://user-images.githubusercontent.com/3610528/77763340-d1472d80-7032-11ea-97fa-3bb792de7ed2.png)